src/async-input.js

```diff
@@ -58,6 +58,12 @@ export const AsyncInputAttributes = {
   NAME: 'name',
 };
 
+export const SUBMIT_TIMEOUT_TYPE = Object.freeze({
+  REGULAR : 10000,
+  /* 5 minutes */
+  INCREASED : 300000
+});
```

Comment:
This should probably live in amp-form.js.

Reply:
wouldn't it make more sense to keep these timeouts and the class names in the async input , since it is only relevant to the async input